tests/manual/showhide/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+[package]
+name = "showhide"
+description.workspace = true
```

Comment:
I think it can have its own description.

Also `publish = false`


In general i'm a bit torn about the value of this test.
I think what'd make sense would be a more generic test that one can run and see many things like initial size, maximum/minimum size, hiding, and so on.

I don't think anyone would otherwise run this kind of test anyway.

Reply:
I think we need some test to verify quickly and easily that hide works. Initially I had it written in an interactive way, where one window with controls would allow controlling visibility of another. That was less magic, but very difficult to debug as debug output was hard to associate between the control window and the test window.

That said, now that all bugs are fixed 🤣 I can change this to be interactive and allow toggling the flags you mentioned. I’ll extend it.